server/app.js

```diff
@@ -19,14 +21,26 @@ app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
-mongoose.connect(process.env.MONGODB_URI, {
-  useNewUrlParser: true,
-}).then(() => console.log("MongoDB successfully connected"))
+app.use((req, res, next) => {
+  res.setHeader("Access-Control-Allow-Origin", "*");
+  res.setHeader(
+    "Access-Control-Allow-Methods",
+    "OPTIONS, GET, POST, PUT, PATCH, DELETE"
+  );
+  res.setHeader("Access-Control-Allow-Headers", "Content-Type, Authorization");
+  next();
+});
+
+mongoose
+  .connect(process.env.MONGODB_URI, {
+    useNewUrlParser: true
+  })
+  .then(() => console.log("MongoDB successfully connected"))
```

Comment:
While good for an all around use case, our setup (create-react-app proxies directly to our backend port so backend thinks it's same origin) deals with this automatically in dev. If we are deploying, create react app will be bundled and served as static assets from the backend (thus won't trigger CORS as it comes directly from backend).